test/SlowTests/Server/Documents/ETL/Queue/AzureQueueStorageEtlTests.cs

```diff
@@ -0,0 +1,465 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Storage.Queues;
+using Azure.Storage.Queues.Models;
+using Newtonsoft.Json;
+using Raven.Client.Documents.Operations.ConnectionStrings;
+using Raven.Client.Documents.Operations.ETL;
+using Raven.Client.Documents.Operations.ETL.Queue;
+using Raven.Client.Documents.Smuggler;
+using Raven.Client.Exceptions.Sharding;
+using Raven.Client.ServerWide.Operations;
+using Raven.Server.Documents.ETL.Providers.Queue;
+using Raven.Server.Documents.ETL.Providers.Queue.Test;
+using Raven.Server.ServerWide.Context;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+using QueueItem = Raven.Server.Documents.ETL.Providers.Queue.QueueItem;
+
+namespace SlowTests.Server.Documents.ETL.Queue;
+
+public class AzureQueueStorageEtlTests : AzureQueueStorageEtlTestBase
```

Comment:
Don't we need to do any cleanup of queues at the end of each test?

Reply:
I achieved it in this way:
https://github.com/djordjedjukic/ravendb/blob/051ba2272c0ec20d347b3bfd07d9a518a432a845/test/SlowTests/Server/Documents/ETL/Queue/AzureQueueStorageEtlTestBase.cs#L91-L96

Should I change it?

Reply:
I was rather thinking about deleting queues during the dispose of tests

Reply:
When you shut down to azurite everything is deleted.

Do you think we need to delete queues explicitly?

Reply:
In our case we'll rather have Azurite emulator always running (in docker) so the cleanup after tests is necessary

Reply:
done